edi-tools/modules/x12xsd/x12xsd.bal

```diff
@@ -161,7 +161,7 @@ function convertSegment(string segmentName, xml x12xsd) returns edi:EdiSegSchema
             fieldSchema.required = minOccurs != "0";
         }
         if conditionalFeildsMap.length() > 0 {
-            string[] nameSplit = regex:split( fieldName, "_");
+            string[] nameSplit = r.split(fieldName);
```

Comment:
Shall we use meaningful names for regex pattern or else lets define this inline if only used once
```suggestion
            string[] nameSplit = re `_`.split(fieldName);
```

Reply:
Sure. Changed with [6baf22b](https://github.com/ballerina-platform/edi-tools/pull/25/commits/6baf22ba11d164bd19d0aaf8cecf52efbbd9e080).